eclipse-jdt/eclipse.jdt.debug
# Fix #91: Stack trace detection not working in colored console

## What it does

Removes the ANSI escape sequences before detecting links.

## How to test

See https://github.com/eclipse-platform/eclipse.platform.debug/issues/91#issuecomment-1294733422

The easiest way to reproduce was to create a new project and `println` the stack trace.

Attached a small file reproducing the problem.
I've tested with ANSI Support enabled and disabled, and I've checked that the links work and jump at the proper line.

## Author checklist


[Main.java.zip](https://github.com/eclipse-jdt/eclipse.jdt.debug/files/9895052/Main.java.zip)
